src/run.rs

```diff
@@ -32,7 +36,15 @@ pub(crate) async fn pull_and_run(
     .map_err(|e| anyhow!("error pulling policy {}: {}", uri, e))?;
 
     if let Some(digest) = verified_manifest_digest {
-        verify::verify_local_checksum(&uri, docker_config, sources, digest).await?
+        verify::verify_local_checksum(
+            &uri,
+            docker_config,
+            sources,
+            digest,
+            &sigstore_opts.fulcio_cert,
+            &sigstore_opts.rekor_public_key,
```

Comment:
What I meant here was to use `&sigstore_opts` as a param. We can change that later if we really want to

Reply:
Ah sorry, right. I misunderstood you.